src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -5,20 +5,107 @@
  * Дотриматися основних вимог щодо реалізації мапи (initial capacity, load factor, resize...)
  * За бажанням можна реалізувати інші методи інтрефейсу Map.</p>
  */
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private Entry<K, V>[] table;
+    private int size;
+    private int capacity = (int) (INITIAL_CAPACITY * LOAD_FACTOR);
```

Comment:
try don't use this variable

Reply:
I used this variable in method rehash when I change my table capacity.. so when I get a new table it saved new capacity. and in future, I will compare the size with a new value of capacity.